### PR TITLE
datafeeder - preliminary work on integrating df deployment

### PR DESCRIPTION
**Pull request by pmauduit**
* unfinished & roughly tested for now, "almost" works out of the box
* will require georchestra/datadir#211 but I'm not sure of the
  conventions to adopt for now ... (port 8080 for the spring boot app ?)
* will require a CI, for now both artifacts are generated in a one-shot
  fashion and uploaded onto packages.g.o/datafeeder/

### CODE REVIEW

**Comment by landryb** [comment]
isnt there a sql schema file to import like all the other schemas (cf `roles/postgres/tasks/other_schemas.yml`) ?
i agree that 8080 will probably conflict, at least with a default still running tomcat..

isnt the hibernate sequence supposed to be created by the hibernate initialization ?

**Comment by pmauduit** [comment]
> isnt there a sql schema file to import like all the other schemas (cf roles/postgres/tasks/other_schemas.yml) ?

here it is, I thought it was okay for 3 SQL lines to put everything into the playbook, and I could not find an "other_schema" variable on top of the georchestra.yml playbook:
https://github.com/georchestra/georchestra/blob/master/postgresql/120-datafeeder.sql


> i agree that 8080 will probably conflict, at least with a default still running tomcat..

any suggestion ? 8089 ?

> isnt the hibernate sequence supposed to be created by the hibernate initialization ?

runtime tested, it seemed that the schema was not sufficient: the backend was complaining about the missing sequence.

**Comment by landryb** [comment]
> > isnt there a sql schema file to import like all the other schemas (cf roles/postgres/tasks/other_schemas.yml) ?
> 
> here it is, I thought it was okay for 3 SQL lines to put everything into the playbook, and I could not find an "other_schema" variable on top of the georchestra.yml playbook:
> https://github.com/georchestra/georchestra/blob/master/postgresql/120-datafeeder.sql

cf `roles/postgresql/vars/mail.yml` :) adding it there should also take care of the hibernate seq creation..

> > i agree that 8080 will probably conflict, at least with a default still running tomcat..
> 
> any suggestion ? 8089 ?

8480 to keep the logic of 8180 8280 8380 like other tomcats ?

**Comment by pmauduit** [comment]
Okay, I think I got your coments in place with the last commit.

I was able to publish a dataset with only one modification related to fix by hand georchesta/georchestra#3358, and also relocating the GS in the web-ui because the default proxyUrl messes it up.

**Comment by landryb** [comment]
quickly tested on top of a running 20.0.x sandbox i had lying around, seems to do the job ! just move the jar outside of `opt` to `srv/webapps` and i'm fine with merging this !

**Comment by landryb** [comment]
+1 to merge :)

File: Vagrantfile [hunk 1]
```diff
@@ -4,12 +4,6 @@
 # Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
 VAGRANTFILE_API_VERSION = "2"
 
-$script = <<SCRIPT
-echo configuring ssh in guest...
-sudo sed -i '/AcceptEnv/d' /etc/ssh/sshd_config
-sudo /etc/init.d/ssh restart
-SCRIPT
```
Comment (fvanderbiest) [inline Vagrantfile:11]: Sigh - There was a reason for this ... do not remember though.

**Pull request merged**